provider/testdata/data/baked/mod.rs

```diff
@@ -20,2087 +20,2890 @@ mod time_zone;
+                const COLLATIONFALLBACKSUPPLEMENTV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_provider_adapters::fallback::provider::CollationFallbackSupplementV1Marker::KEY.hashed();
+                const LOCALEFALLBACKLIKELYSUBTAGSV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_provider_adapters::fallback::provider::LocaleFallbackLikelySubtagsV1Marker::KEY.hashed();
+                const LOCALEFALLBACKPARENTSV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_provider_adapters::fallback::provider::LocaleFallbackParentsV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGDAYRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongDayRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGHOURRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongHourRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGMINUTERELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongMinuteRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGMONTHRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongMonthRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGQUARTERRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongQuarterRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGSECONDRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongSecondRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGWEEKRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongWeekRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const LONGYEARRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::LongYearRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const NARROWDAYRELATIVETIMEFORMATDATAV1MARKER: ::icu_provider::DataKeyHash =
+                    ::icu_relativetime::provider::NarrowDayRelativeTimeFormatDataV1Marker::KEY.hashed();
+                #[cfg(feature = "icu_relativetime")]
+                const NARROWHO
```

Comment:
Suggestion: We don't really need to call this by default, either. It's kind-of elegant if the file defines two macros for you and nothing else.

Reply:
Hmm maybe

Reply:
It's a breaking change to not expose `BakedDataProvider` anymore.

Reply:
It's also a breaking change that `any.rs` doesn't work anymore

Reply:
True. Are we okay with this breakage? Othewise `any.rs` could contain the macro invocation.

Reply:
How about if we add back `any.rs` with a single line to invoke the macro and then make a 2.0-breaking issue to clean this up